packages/tools/devtools/devtools-core/src/messaging/devtools-messages/DevtoolsDisposed.ts

```diff
@@ -0,0 +1,44 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IDevtoolsMessage } from "../Messages";
+
+/**
+ * Encapsulates types and logic related to {@link DevtoolsDisposed.Message}.
+ *
+ * @internal
+ */
+export namespace DevtoolsDisposed {
+	/**
+	 * {@link DevtoolsFeatures.Message} {@link IDevtoolsMessage."type"}.
+	 *
+	 * @internal
+	 */
+	export const MessageType = "DEVTOOLS_DISPOSED";
+
+	/**
+	 * Outbound message broadcasting that the devtools instance in the webpage is terminating.
```

Comment:
Nit: How about something like:
```suggestion
	 * Outbound message indicating that the {@link IFluidDevtools} has been disposed.
```